### ISSUE TITLE
Paths Inside of CSS Files not contain a versionned URL

### ISSUE THREAD

**Issue by seb-jean** [SymfonyCasts/tailwind-bundle#10]
With this bundle, CSS files do not contain a versionned URL.

This section https://symfony.com/doc/current/frontend/asset_mapper.html#paths-inside-of-css-files, works without this bundle.
But it doesn't work when I reinstall the bundle.

Example : This code remains the same.
```css
/* assets/styles/app.css */
.quack {
    background-image: url('../images/duck.png');
}
```

**Comment by weaverryan** [comment]
Fixed in #11 - you're on fire with the bug reports! Thank you!

**Comment by seb-jean** [comment]
it works!
Thx
